release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1056,6 +1056,44 @@ Kubernetes 1.29 removed the following deprecated APIs, so you must migrate manif
 [id="ocp-4-16-cloud-compute-bug-fixes_{context}"]
 ==== Cloud Compute
 
+* Previously, a node associated with a rebooting machine briefly having a status of `Ready=Unknown` triggered the `UnavailableReplicas` condition in the Control Plane Machine Set Operator. This condition caused the Operator to enter the `Available=False` state and trigger alerts because that state indicates a nonfunctional component that requires immediate administrator intervention. This alert should not have been triggered for the brief and expected unavailabilty while rebooting. With this release, a grace period for node unreadiness is added to avoid triggering unnecessary alerts. (link:https://issues.redhat.com/browse/OCPBUGS-34970[*OCPBUGS-34970*])
+
+* Previously, a transient failure to fetch bootstrap data during machine creation, like a transient failure to connect to the API server, caused the machine to enter a terminal failed state. With this release, failure to fetch bootstrap data during machine creation is retried indefinitely until it eventually succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-34158[*OCPBUGS-34158*])
+
+* Previously, the `machine-api` operator panicked when deleting a server in an error state because it was not passed a port list. With this release, deleting a machine stuck in an `ERROR` state does not crash the controller anymore. (link:https://issues.redhat.com/browse/OCPBUGS-34155[*OCPBUGS-34155*])
```

Comment:
```suggestion
* Previously, the `machine-api` operator panicked when deleting a server in an error state because it was not passed a port list. With this release, deleting a machine stuck in an `ERROR` state does not crash the controller. (link:https://issues.redhat.com/browse/OCPBUGS-34155[*OCPBUGS-34155*])
```